pod/perldelta.pod

```diff
@@ -407,7 +1632,7 @@ died, add a short obituary here.
 
 XXX Generate this with:
 
-  perl Porting/acknowledgements.pl v5.37.11..HEAD
+  perl Porting/acknowledgements.pl v5.37.0..HEAD
 
```

Comment:
Shouldn't that line read `perl Porting/acknowledgements.pl v5.39.0..HEAD`?

Reply:
Yes.  This text is from some autogeneration code, which tends to not be good at blead-final releases.

Reply:
No, it is correct.

This document is the v5.38.0 delta, so should credit everyone with changes since v5.36.0 or v5.37.0.  As I am the only person in the acknowledgements for v5.37.0, it doesn't matter which we pick.